Lib/copy.py

```diff
@@ -195,8 +184,11 @@ def _deepcopy_list(x, memo, deepcopy=deepcopy):
     for a in x:
         append(deepcopy(a, memo))
     return y
+
 d[list] = _deepcopy_list
 
+_nil = object()
```

Comment:
Is this used anywhere? If not, can we just keep this region as it is? Less code to review for the core dev.

Reply:
That can indeed be removed, is was added accidentally.

We can also remove the `_nil` argument from `deepcopy` and use a plain `None` as sentinel (keys in the memo are always ints).